### ISSUE TITLE
Unable to preselect first option

### ISSUE THREAD

**Issue by lotif** [hummerichsander/streamlit-free-text-select#2]
First of all, thanks for the awesome component!

One issue I found: if I set `index=0` no option is selected by default. I expect the first option in the list to be selected by default. If I set any other value for index it works as expected. Below is my code:

```python
        model_id = st_free_text_select(
            label="Model ID:",
            options=[
                "meta-llama/Meta-Llama-3-8B-Instruct",
                "meta-llama/Meta-Llama-3-70B-Instruct",
                "meta-llama/Llama-2-7b-chat-hf",
                "meta-llama/Llama-2-13b-chat-hf",
                "meta-llama/Llama-2-70b-chat-hf",
                "mistralai/Mistral-7B-Instruct-v0.2",
                "mistralai/Mistral-7B-Instruct-v0.3",
            ],
            index=0,
        )
```
I expect the component to have `meta-llama/Meta-Llama-3-8B-Instruct` pre-populated but I get blank.

Let me know if you need more info to reproduce. Thanks!

**Comment by hummerichsander** [comment]
Thanks for pointing out this bug. I just fixed it in the latest release 0.1.2.

**Comment by lotif** [comment]
It works great now, thanks @hummerichsander !
